packages/ui/components/editor/plugins/ToolbarPlugin.tsx

```diff
@@ -474,7 +474,7 @@ export default function ToolbarPlugin(props: TextEditorProps) {
               StartIcon={Italic}
               onClick={() => {
                 editor.dispatchCommand(FORMAT_TEXT_COMMAND, "italic");
-                if (isItalic) {
+                if (isBold) {
```

Comment:
@Inderjit-fullstack-dev maybe we can remove these if conditions from this onClick function and https://github.com/calcom/cal.com/blob/6e55db3970efe44b3606cfb9658031b05f30ea41/packages/ui/components/editor/plugins/ToolbarPlugin.tsx#L462

Reply:
Agreed with @Udit-takkar 
I don't see any issue with removing the bold check from italic and italic check from bold. Though there has to be some reason for adding that. 

Blocking it till @CarinaWolli confirms.